pkg/storage/store.go

```diff
@@ -118,11 +119,11 @@ args{
 */
 type args map[string]any
 
-// selectNamed is a shortcut for using `sqlx` Select (SelectContext) with named arguments to prevent writing out the prepare step each time
-func selectNamed(ctx context.Context, np sqlutils.NamedPreparer, dest interface{}, sqlStatement string, arg args) error {
+// namedSelect is a shortcut for using `sqlx` Select (SelectContext) with named arguments to prevent writing out the prepare step each time
+func namedSelect(ctx context.Context, np sqlutils.NamedPreparer, dest interface{}, sqlStatement string, arguments any) error {
```

Comment:
Can we do `interface{}` instead of `any` to match with the underlying methods? 

```suggestion
func namedSelect(ctx context.Context, np sqlutils.NamedPreparer, dest interface{}, sqlStatement string, arguments interface {}) error {
```

Reply:
8e2733b22

A bit different from this suggestion, but @mynar7 and I chatted offline about this change!